ts/model_service_worker.py

```diff
@@ -129,6 +129,11 @@ def load_model(self, load_model_request):
             logging.debug("Model %s loaded.", model_name)
 
             return service, "loaded model {}".format(model_name), 200
+        except Exception as ex:  # pylint: disable=broad-except
```

Comment:
Are naked excepts safe here?

Reply:
@msaroufim Sorry, I didn't get it. Could you please tell me the suggested change? Do you mean I should switch the order?

Reply:
Reversed the order

Reply:
So thing is this except statement will trigger on everything and not just the intended errors. I'd suggest a Runtime error instead

Reply:
Done